manifests/v1beta1/installs/katib-with-kubeflow/kustomization.yaml

```diff
@@ -11,18 +11,16 @@ resources:
 images:
   - name: docker.io/kubeflowkatib/katib-controller
     newName: docker.io/kubeflowkatib/katib-controller
-    newTag: latest
+    newTag: v0.17.0
   - name: docker.io/kubeflowkatib/katib-db-manager
     newName: docker.io/kubeflowkatib/katib-db-manager
-    newTag: latest
+    newTag: v0.17.0
   - name: docker.io/kubeflowkatib/katib-ui
     newName: docker.io/kubeflowkatib/katib-ui
-    newTag: latest
-
-patchesStrategicMerge:
-  - patches/remove-namespace.yaml
+    newTag: v0.17.0
```

Comment:
We should keep `latest` tag in the `master` branch, since we update it only for the release branch:
https://github.com/kubeflow/katib/blob/master/scripts/v1beta1/release.sh#L71

Reply:
Done  :-)